src/pages/Homepage/components/Hero.jsx

```diff
@@ -49,31 +49,35 @@ const [record, setRecord] = useState({
                                 aria-hidden="true"
                                 className="pointer-events-none lg:absolute lg:inset-y-0 lg:max-w-7xl lg:mx-auto lg:w-full"
                             >
-                                <div className="mt-10 absolute transform sm:left-1/2 sm:top-0 sm:translate-x-8 lg:left-1/2 lg:top-1/2 lg:-translate-y-1/2 lg:translate-x-8">
+                                <div
+                                    className="mt-10 absolute transform sm:left-1/2 sm:top-0 sm:translate-x-8 lg:left-1/2 lg:top-1/2 lg:-translate-y-1/2 lg:translate-x-8">
                                     <div className="mt-10 flex items-center space-x-6 lg:space-x-8">
                                         <div className="flex-shrink-0 grid grid-cols-1 gap-y-6 lg:gap-y-8">
-                                            <div className="w-56 h-80 rounded-lg overflow-hidden sm:opacity-0 lg:opacity-100">
+                                            <div
+                                                className="w-56 h-80 rounded-lg overflow-hidden sm:opacity-0 lg:opacity-100">
                                                 {record.title1 ? null : (
-                                                    <Skeleton animation="wave" variant="rectangular" style={{height: '100%'}}/>
-                                                    )}
-                                                    <img
-                                                        style={record.title1 ? {} : {display: 'none'}}
-                                                        src={HeroSection1Images.img1}
-                                                        onLoad={() => onLoadChange("title1")}
-                                                        alt={HeroSection1Images.title1}
+                                                    <Skeleton animation="wave"
+                                                              variant="rectangular" style={{height: '100%'}}/>
+                                                )}
+                                                <img
+                                                    style={record.title1 ? {} : {display: 'none'}}
+                                                    src={HeroImages.img1}
+                                                    onLoad={() => onLoadChange("title1")}
+                                                    alt={HeroImages.title1}
                                                     className="w-full h-full object-center object-cover"
-                                                    />
-                                                    )}
+                                                />
```

Comment:
Codacy has a fix for the issue: [Delete `························`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
                        />
```